usr.sbin/Makefile

```diff
@@ -184,8 +184,11 @@ SUBDIR.${MK_OPENSSL}+=	keyserv
 SUBDIR.${MK_OPENSSL_KTLS}+=	rpc.tlsclntd
 SUBDIR.${MK_OPENSSL_KTLS}+=	rpc.tlsservd
 SUBDIR.${MK_PF}+=	ftp-proxy
-SUBDIR.${MK_PKGBOOTSTRAP}+=	pkg
+# Install pkg64 and/or pkg64c instead of pkg.
 SUBDIR.${MK_PKGBOOTSTRAP}.${MK_LIB64}+=	pkg64
+.if ${MACHINE_ABI:Mpurecap}
+SUBDIR.${MK_PKGBOOTSTRAP}+=	pkg64c
+.endif
```

Comment:
Don't we want to use `pkg` here if we're not targeting purecap?

Reply:
Since we don't care about the hybrid world (as I was told), I guess it would be better to have consistent pkg names across CheriBSD variants to use the same documentation snippets.

Also, in theory, a user could obtain a CheriABI pkg version on a hybrid system.

Reply:
They would need to be built for local64c, which we’re never going to do. We don’t care about hybrid in the sense that we don’t care if it’s no more functional than legacy non-CHERI, but we don’t want to break those or do strange things to them.

Reply:
That is, on a hybrid system having pkg just fetch aarch64 packages and no way to get purecap packages is the “don’t care” position

Reply:
> That is, on a hybrid system having pkg just fetch aarch64 packages and no way to get purecap packages is the “don’t care” position 

In order to support that, we'd have to have a separate package repository with pkg that uses pkg directories without the "64" suffix. An aarch64 ports-mgmt/pkg package has these paths hardcoded.

> They would need to be built for local64c, which we’re never going to do. 

Why they'd need to build for local64c and not use local for CheriABI packages? Isn't local free to use on a hybrid system since aarch64 packages use local64, assuming we don't want to build the third package repository with aarch64 packages using local?

Reply:
> > That is, on a hybrid system having pkg just fetch aarch64 packages and no way to get purecap packages is the “don’t care” position
>
> In order to support that, we'd have to have a separate package repository with pkg that uses pkg directories without the "64" suffix. An aarch64 ports-mgmt/pkg package has these paths hardcoded.

Which is the same as cheribsd-aarch64.

> > They would need to be built for local64c, which we’re never going to do.
>
> Why they'd need to build for local64c and not use local for CheriABI packages? Isn't local free to use on a hybrid system since aarch64 packages use local64, assuming we don't want to build the third package repository with aarch64 packages using local?

Because /usr/local is for the native ABI, which is non-CHERI/hybrid, not purecap.

Reply:
> Which is the same as cheribsd-aarch64.

Wouldn't be it more useful and easier to use the same aarch64 packages as for both hybrid and purecap worlds?

> Because /usr/local is for the native ABI, which is non-CHERI/hybrid, not purecap.

I understand that /usr/local should be for the native ABI and it would be more consistent with upstream. I guess we should decide on whether we want to use the same aarch64 packages on both hybrid and purecap systems first to decide on what `/usr/local` should be for (it could be also a symlink to `/usr/local64` which we tried to avoid for purecap `/usr/local` -> `/usr/local64c`).

Reply:
I changed the code to install only pkg if world is hybrid.